src/ontology/edits/oeo-physical.omn

```diff
@@ -1100,7 +1103,7 @@ Class: GeothermalPowerplant
         <http://purl.obolibrary.org/obo/BFO_0000051> some SteamTurbine
     
     
-Class: GlobalWarmingPotential
+Class: GlobalWarmingPotentialValue
 
     Annotations: 
```

Comment:
Same question like Power/PowerValue: GlobalWarmingPotential and GlobalWarmingPotentialValue is not the same in my opinion.
What about implementing a property 'has_quantityValue', something like that?